src/main/java/se/sundsvall/messaging/model/DeliveryMode.java

```diff
@@ -0,0 +1,5 @@
+package se.sundsvall.messaging.model;
+
+public enum DeliveryMode {
```

Comment:
Den här skulle jag nog flytta in i t.ex. `LetterRequest`, eftersom det kommer att komma in en annan ´DeliveryMode` från regeltjänsten så småningom, och just den här `DeliveryMode`:n är väldigt letter-specifik